indicators/fake-crypto-mining-ReceiveVoucher4.yml

```diff
@@ -0,0 +1,43 @@
+title: Fake crypto mining - ReceiveVoucher4
+
+description: |
+    Detects a malicious DApp that pretends to be a liquidity mining platform
+    while presenting fake audit reports and partners. AJAX call to receive
+    other contents.
+
+references:
+    - https://urlscan.io/search/#hash%3A5218075e53acd18fdd4b8c92cb0dad377bffab9d072cc6205e51ee8e32514a14
+    - https://urlscan.io/result/011814b3-f30a-4ac7-bac1-db62f71c29de/
+    - https://urlscan.io/result/07c863ae-f341-4908-9686-2fc030c40a47/
+    - https://urlscan.io/result/a43482f4-83ee-4f17-9928-f6c4bc437e2f/
+
+related:
+    - fake-crypto-mining-ReceiveVoucher
+
+detection:
+
+    obfuscatedJS:
```

Comment:
Considering the deobfuscated JS looks something like
```js
$.post('user_ajax.php', { type: 'get_html' }, function (_0x112088, _0x2aa5d2) {
  $('body').html(_0x112088)
})
```

I would suggest detecting the strings that are shared, so that the rule canvases both types of the JS

Reply:
Gotcha, thanks!